File: Golang/File2Go/file2.go
package main

import "fmt"

func FindLargest(anyArray []int) int{

	largest:= anyArray[0];
	for i:= 0; i < len(anyArray); i++ {
		if anyArray[i] > largest{
			largest = anyArray[i]
		}
	}
	return largest
}

func main() {
	arr := []int {2,4,6,8,90,45,-25};
	result := FindLargest(arr)
	fmt.Println("The largest NUmber in an array :", result)
}

File: Java/src/Day_4/File2.java
package Day_4;

public class File2 {

    public static int FindLargest(int arr[]){
        int largest = arr[0];
        for (int i=0; i<arr.length; i++){
            if(arr[i] > largest){
                largest = arr[i];
            }
        }
        return largest;
    }
    public static void main(String[] args){
        int arr[] = {1,2,3,4,5,533,22,78,343,643,-56,89,244};
        int result = FindLargest(arr);
        System.out.println("The largest Number in an array is :"+" "+result);
    }
}
